metric_learn/base_metric.py

```diff
@@ -154,4 +255,5 @@ def score(self, quadruplets, y=None):
     score : float
       The quadruplets score.
     """
-    return - np.mean(np.sign(self.decision_function(quadruplets)))
+    quadruplets = check_tuples(quadruplets)
+    return - np.mean(self.predict(quadruplets))
```

Comment:
The space between the `-` and `np.mean` is a little confusing.

Reply:
Agreed
Done